lib/active_storage_validations/dimension_validator.rb

```diff
@@ -120,10 +127,10 @@ def is_valid?(record, attribute, file_metadata)
       true # valid file
     end
 
-    def add_error(record, attribute, type, **attrs)
-      key = options[:message].presence || type
-      return if record.errors.added?(attribute, key)
-      record.errors.add(attribute, key, **attrs)
+    def add_error(record, attribute, default_message, **attrs)
+      message = options[:message].presence || default_message
```

Comment:
Maybe an interesting possibility is to allow a proc for the message as well. But lets do that in a separate PR if people would like that idea.